origin-dapp/src/utils/listing.js

```diff
@@ -62,6 +61,27 @@ export function dappFormDataToOriginListing(formData) {
   return listingData
 }
 
+function getCategory(category) {
+  switch(category) {
+    case 'schema.housing':
+    case 'schema.tickets':
+      return 'forSale'
+    default:
+      return category
+  }
+}
+
+function getSubCategory(subCat) {
+  switch(subCat) {
+    case 'schema.housing.vacationRentals':
```

Comment:
I think there are a few more subcats that have legacy listings and are being deprecated.
See this xsheet with data I queried from prod (look at 3rd tab): https://docs.google.com/spreadsheets/d/1K7ZGnEqLJxPM9MI27sjr4m_U9yyA5DFdylmzOxJq04M/edit#gid=1600462284